solo-chains/runtime/starlight/src/genesis_config_presets.rs

```diff
@@ -306,6 +310,14 @@ fn starlight_testnet_genesis(
 
     const ENDOWMENT: u128 = 1_000_000 * STAR;
 
+    let core_percentage_for_pool_paras = Perbill::from_percent(100).saturating_sub(
```

Comment:
Right now registering a parathread in the starlight is not allowed, so maybe we can put this to 100% for parachains?

Reply:
we are leaving it dinamically finally after internal discussion

Reply:
disregard